core/src/smartcontracts/isi/tx.rs

```diff
@@ -7,6 +7,16 @@ use iroha_telemetry::metrics;
 
 use super::*;
 
+impl<W: WorldTrait> ValidQuery<W> for FindAllTransactions {
+    #[log]
```

Comment:
please don't use `#[log]`. It causes deadlocks as far as I'm aware of

Reply:
Doesnt matter, I'll remove it in #2186 anyway.

Reply:
I'm think @appetrosyan PR got merged first, so this would reintroduce the `#[log]`